src/main/java/camp/nextstep/edu/calculator/StringCalculator.java

```diff
@@ -0,0 +1,85 @@
+package camp.nextstep.edu.calculator;
+
+import camp.nextstep.edu.calculator.exception.NegativeNumberException;
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.springframework.util.StringUtils;
+
+public class StringCalculator {
+
+	private static final String DELIMITER = ",|:";
```

Comment:
들여쓰기에는 탭 문자가 사용되지 않습니다. 아래의 Google Java Style Guide를 참고해 주세요.

> Tab characters are not used for indentation.

```suggestion
    private static final String DELIMITER = ",|:";
```